src/bot/commands/AvatarCommand.ts

```diff
@@ -22,7 +23,9 @@ export default class AvatarCommand implements UserCommand {
 
   public run(message: Message, params: string[]) {
     if (!message.member) return;
-    if (!message.member.hasPermission(Permissions.FLAGS.ADMINISTRATOR!)) return;
+
+    const allowedToRunCommand = userHasElevatedRole(message.member.roles.cache);
+    if (!message.member.hasPermission(Permissions.FLAGS.ADMINISTRATOR!) && !allowedToRunCommand) return;
```

Comment:
From the original comment

> The way you implemented it, the user would have to be in the rolesAllowedToRemoveSounds AND they would still need to have the administrator flag. That does not change the functionality all too much, and maybe that is a bit too overkill. We are allowing users to set the roles explicitly, so I don't think we need the extra check anymore. What do you think?

Looking at it now, we could either
1. Always just rely on `userHasElevatedRole(message.member.roles.cache);`, so 

```ts
if (userHasElevatedRole(message.member.roles.cache)) {
  return;
}
```

2. _Even if we still want to_ check for the `Permissions.FLAGS.ADMINISTRATOR` flag, we could move that logic into the function as well, right?

Reply:
I moved the `Permissions.FLAG.ADMINISTRATOR` flag into the `userHasElevatedRole` method